README.md

```diff
@@ -102,39 +99,90 @@ func main() {
+Writers can be configured through `spectator.Config.Location` (`sidecar.output-location` in file based configuration).
+Possible values are:
 
-```
-level=error msg="Could not POST measurements: HTTP 200 context canceled"
-level=error msg="Unable to close body: context canceled"
-```
+- `none`
+- `stdout`
+- `stderr`
+- `memory`
+- `file:///path/to/file`
+- `unix:///path/to/socket`
+- `udp://host:port`
+
+Location can also be set through the environment variable `SPECTATOR_OUTPUT_LOCATION`. If both are set, the Config value
+takes precedence over the environment variable.
+
+#### Meters
+
+The following new Meters have been added:
+
+- `meter.MaxGauge`
+- `meter.Gauge` with TTL
+
+#### Common Tags
+
+Common tags are now automatically added to all Meters. Their values are read from the environment variables.
+
+| Tag          | Environment Variable |
+|--------------|----------------------|
+| nf.container | TITUS_CONTAINER_NAME |
+| nf.process   | NETFLIX_PROCESS_NAME |
+
+### Moved
+
+- Runtime metrics collection has been moved
+  to [spectator-go-runtime-metrics](https://github.com/Netflix/spectator-go-runtime-metrics). Follow instructions in
+  the [README](https://github.com/Netflix/spectator-go-runtime-metrics) to enable
+  collection.
+- Some types have been moved to different packages. For example, `spectator.Counter` is now in `meter.Counter`.
+
+### Removed
+
+- `spectator.HttpClient` has been removed. Use the standard `http.Client` instead.
+- `spectator.Meter`s no longer has a `Measure() []Measurement` function. Meters are now stateless and do not store
+  measurements.
+- `spectator.Clock` has been removed. Use the standard `time` package instead.
+- `spectator.Config` has been greatly simplified.
+    - If you're using file based configuration, `"common_tags"` has been renamed to `"sidecar.common-tags"`.
+- `spectator.Registry` no longer has a `Start()` function. It is now automatically started when created.
+- `spectator.Registry` no longer has a `Stop()` function. Instead, use `Close()` to close the registry.
+- `spectator.Config.IpcTimerRecord` has been removed. Use a `meter.Timer` instead to record Ipc metrics.
+- `spectator.MeterFactoryFun` has been removed. If you need to create a custom meter you can do so by wrapping one of
+  the meters returned by `spectator.Registry`.
+- `spectator.Registry` no longer reports `spectator.measurements` metrics. Instead, you can use spectatord metrics to
+  troubleshoot.
+- `spectator.Registry` no longer keep track of the Meters it creates. This means that you can't get a list of all Meters
+  from the Registry. If you need to keep track of Meters, you can do so in your application code.
+- `Percentile*` meters no longer support defining min/max values.
+
+### Migration steps
 
-[Issue#49366]: https://github.com/golang/go/issues/49366
-[PR#59]: https://github.com/Netflix/spectator-go/pull/59
+1. Make sure you're not relying on any of the [removed functionality](#removed).
```

Comment:
We need to add a step to mention that any dependencies on the internal configuration library should be removed.

Reply:
What configuration library are you referring to?